convert_util.h

```diff
@@ -71,6 +71,10 @@ struct convert_opts {
 	/* if CONVERT_F_ERROR is set in flags */
 	uint8_t		error_code;
 
+	/* if CONVERT_F_EXTENDED_TCP_HDR is set in flags
```

Comment:
You could do something like:

```
uint8_t *options;
size_t n_options;
```

options needs to be malloc'ed and free'd by the caller.

Reply:
Done for writing. Will add support for reading in another commit.